packages/keystatic/src/form/fields/markdoc/editor/schema.tsx

```diff
@@ -315,27 +279,6 @@ const nodeSpecs = {
     toDOM() {
       return blockquoteDOM;
     },
-    reactNodeView: {
```

Comment:
This is the only part i'm unsure about. Removing `reactNodeView` didn't seem to affect editor rendering but i'm definitely missing a bunch of context.

The wrapping `<div>` breaks margin expectations, though that's more of a shortcoming with the prose styles.